### PR TITLE
Added Sketch file and steps to open Sketch file on local PC

### PR DESCRIPTION
**Pull request by victoriali8023**

### CODE REVIEW

**Comment by pushyamig** [comment]
I was able to Open the `*.sketch` files in Adobe XD. I don't necessarily want to do stuff in Figma, but thanks for adding that info too. If we get to point that we need an account on Figma we can comeback to it

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
 # TL Projects UX Work
+Two ways to open Sketch file in your local laptop:
```
Comment (pushyamig) [inline README.md]: @victoriali8023 Let's add the Readme from the top-level to ccm Readme. The info provided here is generic but I am not sure yet how the repo evolves for UX work and what tools they prefer.

Comment (victoriali8023) [reply README.md]: Okay, I will revise the my Readme. As for the future tool, Adobe XD and Sketch may still be two main options for designers (if Figma is not secure).

File: README.md [hunk 2]
```diff
@@ -1 +1,22 @@
 # TL Projects UX Work
+
+## Open UX Work on Adobe XD
```
Comment (pushyamig) [inline README.md:3]: Move the Readme content to https://github.com/tl-its-umich-edu/UX-Work/blob/master/ccm/README.md instead?
CCM folder has another README

File: README.md [hunk 3]
```diff
@@ -1 +1,22 @@
...
+
+2. Install Adobe XD on your local laptop
+3. Open Adobe XD
+4. Click "File"
+5. Choose "open.."
+6. Select "Your computer"
+7. Choose the Sketch file you just downloaded from Github.
+8. Success!
+
+## Not secure - Open UX Work on Figma
```
Comment (pushyamig) [inline README.md]: Remove `Not secure`. The point I wanted to make is I don't want to use Figma now. But maybe in the future. It is OK to have now in README

**Pull request merged**